IdentityCore/src/cache/mac/MSIDMacCredentialCacheItem.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+#import <Foundation/Foundation.h>
+#import "MSIDCredentialCacheItem.h"
+#import "MSIDDefaultCredentialCacheKey.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDMacCredentialCacheItem : NSObject <MSIDJsonSerializable>
```

Comment:
I understand its not a nice name but even the blob manager rename would not solve the issue I believe because it is like combined item of all credentials as well. Think of a case, when you already have a blob in cache, and you need to merge the in memory blob with the persistent blob. If the class name is MSIDMacBlobManager, then what would we rename this method to. Merge Blob or something. We need to come up with a better name:)